packages/reports/tests/acceptance/navi-report-test.js

```diff
@@ -404,6 +404,8 @@ test('Save As report', function(assert) {
       '(New Copy) Hyrule News',
       'New Saved Report is being viewed'
     );
+
+    assert.notOk(find('.save-as__save-as-modal-btn').is(':visible'), 'Save As Modal not visible after save');
```

Comment:
Does this work? I thought `.is` was a jquery function

Reply:
it is because we're still using jquery in tests atm and this is just an inverse of an earlier assertion.  let me see if I can update them both to non jquery version.

Reply:
Ok pushed this so that upgrading will be easier.